openmls/src/ciphersuite/aead.rs

```diff
@@ -1,15 +1,33 @@
+use std::ops::Deref;
+
+use zeroize::ZeroizeOnDrop;
+
 use super::*;
 
 /// The default NONCE size in bytes.
 pub(crate) const NONCE_BYTES: usize = 12;
 
+/// Wrapper on top of a byte vector to implement ZeroizeOnDrop.
+#[derive(Serialize, Deserialize, ZeroizeOnDrop)]
+#[cfg_attr(any(feature = "test-utils", test), derive(Clone, PartialEq, Eq))]
+#[cfg_attr(feature = "crypto-debug", derive(Debug))]
+pub struct AeadValue(Vec<u8>);
```

Comment:
Implement `impl From<Vec<u8>> for AeadSecretKey {}`

Reply:
I don't think that's necessary
it is constructed only once and it is in the `AedKey`.